crates/aggchain-proof-core/src/proof.rs

```diff
@@ -124,3 +96,8 @@ pub struct AggchainProofPublicValues {
     /// Chain-specific commitment forwarded by the PP.
     pub aggchain_params: Digest,
 }
+
+/// Helper function to compute the commitment for global indices.
+pub fn compute_commit_imported_bridge_exits(global_indices: &[B256]) -> Digest {
+    keccak256_combine(global_indices.iter().map(|idx| keccak256(idx.as_slice())))
+}
```

Comment:
should we also do the same switch as https://github.com/agglayer/agglayer/pull/721 directly in this PR?

Reply:
oki will add it on this PR as well ^^